crates/extensions/aws_mapper_ext/src/converter.rs

```diff
@@ -49,15 +44,16 @@ impl AwsConverter {
     }
 }
 
-#[async_trait]
+pub fn make_valid_topic_or_panic(topic_name: &str) -> Topic {
+    Topic::new(topic_name).expect("Invalid topic name")
+}
```

Comment:
This function is useless. Use instead  `Topic::new_unchecked()` that does the same.